CRM/MembershipExtras/Service/MembershipEndDateCalculator.php

```diff
@@ -9,15 +12,68 @@ class CRM_MembershipExtras_Service_MembershipEndDateCalculator {
    * @param int $membershipId
    *
    * @return string
+   * @throws \CiviCRM_API3_Exception
+   * @throws \CiviCRM_API3_Exception|\Exception
    */
   public static function calculate($membershipId) {
```

Comment:
Some if the methods use $membershipId, some use $membershipID . I understand that they can define differently but should it be consistent for the entire class?